.github/workflows/workflow.yaml

```diff
@@ -374,10 +374,11 @@ jobs:
             echo "::set-output name=terraformdocsconfig::.terraform-docs.yaml"
           else
             echo "Using workflow configuration for commitlint"
-            echo "::set-output name=terraformdocsconfig::.github-workflow-terraform-config/.terraform-docs.yaml"
+            echo "::set-output name=terraformdocsconfig::/home/runner/work/.github-workflow-terraform-config/.terraform-docs.yaml"
```

Comment:
This is not a static path. Use `$GITHUB_WORKSPACE` for a dynamic absolute path.

Reply:
https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
den er jo `/home/runner/work/my-repo-name/my-repo-name`. så det vil vel ikke fungere så bra siden jeg vil at den skal sjekke ut på utsiden?

Reply:
Men plutselig kan denne kjøre på en runner som kjører med et annet brukernavn, eller annet environment. Om du vil ha noe midlertidig ville jeg heller lent meg på /tmp, og helst med `DIR=$(mktmp -d)`, fordi de ikke er spesielt godt isolert på filystemet.

Reply:
vil et temp dir bli med fra et step til et annet?

Reply:
Ja, mellom steps, men ikke _nødvendigvis_ mellom `job`s. Du må bare outputte pathen slik at det andre stepet vet hvor det er.

Reply:
lagt inn variabelen